samples/server/petstore/java-play-framework-controller-only/public/swagger.json

```diff
@@ -307,7 +306,7 @@
           "type" : "file"
         } ],
         "responses" : {
-          "200" : {
+          "201" : {
```

Comment:
I'm surprised the json changed... If I'm not mistaken, this is simply a dump from swagger-parser in json format. Why is it now using the 201 code instead of 200. Did I miss something?

Reply:
Agreed.

Reply:
my mistake, I did a few tests and I add the file to the commit without notice it